python/ray/_private/ray_constants.py

```diff
@@ -439,6 +439,8 @@ def env_set_by_user(key):
 KV_NAMESPACE_JOB = b"job"
 KV_NAMESPACE_CLUSTER = b"cluster"
 KV_HEAD_NODE_ID_KEY = b"head_node_id"
+# Assure this stays in sync with kGcsPidKey in gcs_server.cc
```

Comment:
You can expose c++ constant to python via cython: https://github.com/ray-project/ray/blob/efc048a46a35c92a86deda4e13b2646f79795ad9/python/ray/includes/common.pxi#L122